tests/migrations/test_state.py

```diff
@@ -46,7 +46,7 @@ class Meta:
                 app_label = "migrations"
                 apps = new_apps
                 unique_together = ["name", "bio"]
-                index_together = ["bio", "age"]
+                indexes = [models.Index(fields=["bio", "age"])]
```

Comment:
This test making a ProjectState with all options. We shouldn't remove `index_together` but mark it for removal when deprecation ends:
```suggestion
                index_together = ["bio", "age"]  # RemovedInDjango51Warning.
```